### ISSUE TITLE
[FEARURE] Add music supoort

### ISSUE THREAD

**Issue by AykhanUV** [anilbeesetti/nextplayer#1120]
Can you add music support?

**Comment by VasilisKos** [comment]
This is a VIDEO player!

**Comment by athretv** [comment]
That's the reason i opened this, to bring music support to app

**Comment by thisisartman** [comment]
> That's the reason i opened this, to bring music support to app

I don't think it makes sense to add music support to an app that is specifically being developed as a video player.
Try using other music players like Metro Music player, a FOSS player.

**Comment by champvikash** [comment]
if all the participants agree i will create  music section particular in this app.

**Comment by thisisartman** [comment]
> if all the participants agree i will create music section particular in this app.

No need to. The app is a great video player that can also play music files. we don't need it to be a music player as well. There are many great OSS music players to choose from. I'm sharing a couple here so people can be satisfied: [Metro](https://f-droid.org/en/packages/io.github.muntashirakon.Music/), [Pulse Music](https://f-droid.org/en/packages/com.hardcodecoder.pulsemusic/), [Music Player Lite](https://f-droid.org/en/packages/com.atul.musicplayer/).

**Comment by champvikash** [comment]
@anilbeesetti close this issue not req. music player in this project.
